libsmtutil/SolverInterface.h

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
Line 59 and 67 doesn't seem to be implemented in the z3/cvc4 files, maybe it is some old leftover?

Reply:
Argh, true... `smtSort` is used exactly by the `Expression` constructor that takes `Type`

Reply:
Maybe I need to remove that and only take `Sort`, and the places that use `Expression(Type)` need to use `Expression(Sort)` instead